packages/mobile/src/components/offline-downloads/OfflineStatusRow.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import type { ID } from '@audius/common/models'
+
+import { CollectionDownloadStatusRow } from 'app/screens/collection-screen/CollectionDownloadStatusRow'
+
+import { TrackDownloadStatusIndicator } from './TrackDownloadStatusIndicator'
+
+interface OfflineStatusRowProps {
+  contentId?: ID
+  isCollection: boolean
+}
+
+export const OfflineStatusRow = (props: OfflineStatusRowProps) => {
+  const { contentId, isCollection } = props
```

Comment:
nit: ditto - why not just destructure props directly in the line above?

Reply:
I know @dylanjeffers prefers this, though it's not in our styleguide. Personally, I don't mind inline if there are few props.

@dylanjeffers care to defend your preference? If so, should we codify it in the guide?